tools/manually-handling-sharding-ddl-locks.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Handle Sharding DDL Locks Manually in TiDB-DM
+summary: Learn how to handle sharding DDL locks manually in TiDB-DM.
+category: tools
+---
+
+# Handle Sharding DDL Locks Manually in TiDB-DM
+
+The sharding DDL lock of TiDB-DM automatically completes the synchronization process in most cases, but under some abnormal scenarios you need the command `unlock-ddl-lock` or `break-ddl-lock` to handle abnormal DDL locks manually.
+
+> **Warning:**
+>- Do not use `unlock-ddl-lock` or `break-ddl-lock` unless you are totally aware of the possible impacts brought by the command and you can accept them.
+>- Before manually handling the abnormal DDL lock, make sure that you have already read the DM [shard merge principles](./shard-merge.md#Principles).
```

Comment:
`shard-merge.md` not added yet?

Reply:
yep, should update shard solution document, ref issue https://github.com/pingcap/docs/issues/874
 and @lilin90 told me we should use absolute path, `/tools/xx/shard_merge.md`

Reply:
Yes, I'm translating it and will commit a PR for it today.